components/search/TierToggler.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <button
+        class="toggle-button"
+        data-qa="tier toggler"
+        @click="toggleHandler"
+      >
+        {{ !toggle ? button.show : button.hide }}
+      </button>.
+    </p>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'TierToggler',
+
+    props: {
```

Comment:
Since the toggle is always for the content tier, it shouldn't be necessary to pass in all these props, they're always going to be the same.
The only thing we should need to pass in is the state/active status, in case we want to initialize the component where tier 0 is already being shown. For the case where someone loads the URL with it active.
The rest of the attributes can be computed properties based on the state.
Perhaps the component needs to know the query applied on the current page in order to generate the `to` link?